source/core/GenericFunctions.cpp

```diff
@@ -1733,9 +1731,29 @@ VIREO_FUNCTION_SIGNATURE1(IsEQAccumulator, void)
         while (ExecutionContext::IsNotCulDeSac(pInstruction)) {
             pInstruction = _PROGMEM_PTR(pInstruction, _function)(pInstruction);
             if (!*dest)
-                return null;
+                return true;
         }
     }
+    return false;
+}
+//------------------------------------------------------------
+// Accumulators are used for elements of an array and elements of a cluster
+VIREO_FUNCTION_SIGNATURE1(IsEQAccumulator, void)
+{
+    BinaryCompareInstruction* binop = (BinaryCompareInstruction*)_ParamPointer(0);
+    if (DoIsEQBinayCompareAccumulator(binop)) {
+        return null;
+    }
+    return _this;
+}
+//------------------------------------------------------------
+// Accumulators are used for elements of an array and elements of a cluster
+VIREO_FUNCTION_SIGNATURE1(IsEQSearchAccumulator, void)
+{
+    BinaryCompareInstruction* binop = (BinaryCompareInstruction*)_ParamPointer(0);
+    if (DoIsEQBinayCompareAccumulator(binop)) {
+        return null;
+    }
```

Comment:
I don't think this wrap is necessary, see my other comment on LTSortAccumulator.